docs/architecture/adr-076-combine-spec-repo.md

```diff
@@ -0,0 +1,97 @@
+# ADR 076: Combine Spec and Tendermint Repositories 
+
+## Changelog
+
+- 2022-02-04: Initial Draft. (@tychoish)
+
+## Status
+
+Accepted.
+
+## Context
+
+While the specification for tendermint was original in the same
+repository as the Go implementation, at some point the specification
+was split from the core tendermint repository and maintained
+separately from the implementation. While this makes sense in
+promoting a conceptual separation of specification and implementation,
+in practice this separation was a premature optimization apparently
+aimed at supporting alternate implementations of tendermint. 
+
+However, maintaining a spec repo has ended up being a premature
+optimization: there are no current projects to develop alternate
+implementations of tendermint based on the common specification, and
+having separate repositories creates an ongoing maintenance burden to
+support versioning and publication.
```

Comment:
Some suggestions to consolidate themes, and to remove duplication of the previous paragraph's conclusion.

```suggestion
The operational and documentary burden of maintaining a separate
spec repo has not returned value to justify its cost. There are no active
projects to develop alternate implementations of Tendermint based on the
common specification, and having separate repositories creates an ongoing
burden to coordinate versions, documentation, and releases.
```